### ISSUE TITLE
Getting blank repo name every time when publishing to AWS Lambda

### ISSUE THREAD

**Issue by M-Imtiaz** [aws/aws-toolkit-visual-studio#163]
**Describe the bug**
When I try to upload lambda function from visual studio, the image repo name always fills blank. I also tried removing aws-lambda-tools-defaults.json and generated again, but the issue didn't resolve. 

**Expected behavior**
The expected behavior should be, image repo name should not be blank after the first time upload. 

**Screenshots**
![image](https://user-images.githubusercontent.com/24789996/122549501-fcf56f80-d04b-11eb-81c7-00b18e59696d.png)

- Windows Version: Windows 10
- Visual Studio Version: 2019 (16.10.2)
- AWS Toolkit for Visual Studio Version: 1.21.2.0

**Comment by awschristou** [comment]
Out of curiosity, are you able to list the ECR repos from your account? I'm wondering if there are permissions preventing that dialog from listing the available repos. If you go to the AWS Explorer, expand "Amazon Elastic Container Service", then expand "Repositories", please let me know if this lists your repos.

**Comment by M-Imtiaz** [comment]
@awschristou Yes, I have access to ECR repos. I have permission. I can also see the list in visual studio when I open the Image Repo dropdown list. My point is, should it not automatically pick the repo name? As repo name is stored in aws-lambda-tools-defaults.json after the first-time upload.

**Comment by shruti0085** [comment]
Thank you for reporting this @M-Imtiaz - I can confirm that I was able to reproduce this.

**Comment by nCubed** [comment]
@awschristou VS 2019 latest AWS toolset / Win 10 x64. I too can confirm this behavior:
* GUI opens and the repo is initially selected in the drop down.
* A moment later and the repo name is cleared.
* Must then manually select the repo from the drop down.

Not too big of a deal, but certainly annoying enough.
